endercore/src/main/java/com/enderio/core/common/menu/SyncedMenu.java

```diff
@@ -80,4 +83,16 @@ public T getBlockEntity() {
     public List<EnderDataSlot<?>> getClientToServerSlots() {
         return clientToServerSlots;
     }
+    
+    public void addInventorySlots() {
```

Comment:
add x/y as parameters so that non default sizes work as well

Reply:
Not sure that would be enough, like the position, padding and inventory. I can try to make it more generic though.

Reply:
position is enough. the slots will have the default layout, but might be a bit higher/lower